discovery-provider/src/tasks/tracks.py

```diff
@@ -219,6 +243,8 @@ def parse_track_event(
                 track_record.cover_art_sizes = track_record.cover_art
                 track_record.cover_art = None
 
+        update_credit_splits(session, track_record, track_metadata)
+
     if event_type == track_event_types_lookup["delete_track"]:
```

Comment:
What happens if a track with credit splits (on either side) is deleted?

Reply:
I think it shouldn't matter really because we don't ever delete records (we just set is_delete).  When parent_track_id gets deleted, it's still reasonable that a child says they "sampled" or "remixed it" and the opposite is probably true too? wdyt?